insights/parsers/tests/test_nmcli.py

```diff
@@ -91,11 +91,6 @@
 enp0s3         bfb4760c-96ce-4a29-9f2e-7427051da943  802-3-ethernet  enp0s3"
 """.strip()
 
-NMCLI_SHOW_ERROR_2 = """
```

Comment:
I think this is a good abnormal test case which should be handled correctly since you have already imagined this kind of situation.

Better Skip it, e.g. when the `self.data` is empty. how do you think?

Reply:
@xiangce  - Done, Added exception check, if data is invalid then it directly throw an error hence added  `try:  <code>  except:<code>` block.